litmus/director/TCID-DIR-OP-INSTALL-OPENEBS/test_vars.yml

```diff
@@ -11,8 +11,8 @@ openebs_components:
     'cspc-operator',
     'cvc-operator',
   ]
-openebs_ds_component: ['openebs-ndm']
-control_plane_node_label: ['mayadata.io/control-plane=true']
+openebs_ds_component: "openebs-ndm"
+control_plane_node_label: "mayadata.io/control-plane=true"
```

Comment:
For string we are not using double quotes .
ref: https://github.com/mayadata-io/oep-e2e/blob/f4c8d51dd57de1d89f9293c2f45de4edb9a99583/litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-CP-ON-SPECIFIC-NODE/test_vars.yml#L14